<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWidget</class>
 <widget class="QWidget" name="MenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 5);</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="gif_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>100</y>
       <width>420</width>
       <height>278</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>330</y>
       <width>420</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 28px &quot;PIXY&quot;;

border: solid;

background-color: rgba(0,  0, 0, 0);

color: black;
text-align: left;</string>
     </property>
     <property name="text">
      <string>A1_MAZE_CPP

Created by
zekkoaem &amp; stranger</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="maze_button">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>495</y>
       <width>420</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: solid;

	background-color: rgba(27,  27, 27, 255);

	color: white;
	text-align: center;

	font: 14px &quot;PIXY&quot;;
}

QPushButton:pressed {
	color: black;
    background-color:  rgba(200, 200, 200, 100);
}</string>
     </property>
     <property name="text">
      <string>Maze</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cave_button">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>470</y>
       <width>420</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: solid;

	background-color: rgba(27,  27, 27, 255);

	color: white;
	text-align: center;

	font: 14px &quot;PIXY&quot;;
}

QPushButton:pressed {
	color: black;
    background-color:  rgba(200, 200, 200, 100);
}</string>
     </property>
     <property name="text">
      <string>Cave</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_button">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>545</y>
       <width>420</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: solid;

	background-color: rgba(27,  27, 27, 255);

	color: white;
	text-align: center;

	font: 14px &quot;PIXY&quot;;
}

QPushButton:pressed {
	color: black;
    background-color:  rgba(200, 200, 200, 100);
}</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
